assets/app/view/game/buy_sell_shares.rb

```diff
@@ -12,114 +13,138 @@ class BuySellShares < Snabberb::Component
       needs :corporation
 
       def render
-        step = @game.round.active_step
-        current_entity = step.current_entity
+        @step = @game.round.active_step
+        @current_entity = @step.current_entity
 
-        ipo_share = @corporation.shares[0]
-        pool_shares = @game.share_pool.shares_by_corporation[@corporation].group_by(&:percent).values.map(&:first)
+        @ipo_shares = @corporation.shares.group_by(&:percent).values
+          .map(&:first).sort { |a, b| b.percent <=> a.percent }
```

Comment:
are you just trying to do a reverse or something?

sort.reverse

or sort_by { |x| -x.percent }

Reply:
Just reverse. Will use the cleaner code.